src/options/error.rs

```diff
@@ -1,37 +1,30 @@
-use std::ffi::OsString;
 use std::fmt;
 use std::num::ParseIntError;
 
-use crate::options::flags;
-use crate::options::parser::{Arg, Flag, ParseError};
-
 
 /// Something wrong with the combination of options the user has picked.
 #[derive(PartialEq, Eq, Debug)]
 pub enum OptionsError {
 
-    /// There was an error (from `getopts`) parsing the arguments.
-    Parse(ParseError),
-
     /// The user supplied an illegal choice to an Argument.
-    BadArgument(&'static Arg, OsString),
+    BadArgument(String, String),
 
     /// The user supplied a set of options that are unsupported
     Unsupported(String),
 
     /// An option was given twice or more in strict mode.
-    Duplicate(Flag, Flag),
+    Duplicate(String, String),
 
     /// Two options were given that conflict with one another.
-    Conflict(&'static Arg, &'static Arg),
+    Conflict(String, String),
 
     /// An option was given that does nothing when another one either is or
     /// isn’t present.
-    Useless(&'static Arg, bool, &'static Arg),
+    Useless(String, bool, String),
 
     /// An option was given that does nothing when either of two other options
     /// are not present.
-    Useless2(&'static Arg, &'static Arg, &'static Arg),
+    Useless2(String, String, String),
```

Comment:
I think we also need a `Useless3` and `Useless4` because ... why not! :D

Reply:
Those where there before me tbh

Reply:
Yeah, I know was just a joke :D

Reply:
Oh my bad sorry